libs/eo/certificates/src/lib/eo-certificates.service.ts

```diff
@@ -26,6 +26,10 @@ export interface EoCertificate {
   dateTo: number;
   quantity: number;
   gsrn: string;
+  techCode: string;
+  fuelCode: string;
+  id: string;
+  gridArea: string;
```

Comment:
Is this used or just prepared for the future?

Reply:
It is used :)